_posts/newsletters/2021-11-30-newsletter.md

```diff
@@ -0,0 +1,138 @@
+
+In this issue:
+
+* [Second Annual General Meeting and Election of Steering Committee Members](#agm)
+* [International RSE Survey](#survey)
+* [US-RSE Community Workshop](#workshop)
+* [US-RSE Community Calls](#community-calls)
+* [RSE Stories Podcast](#podcast)
+* [Recent Job Postings](#jobs)
+* [Community News](#news)
+* [DEI Media Club Discussion](#dei-mc)
+* [Interesting Events](#events)
+* [Interesting Reads](#reads)
+* [US-RSE Outreach](#outreach)
+* [US-RSE Financials](#financials)
+* [Get Involved](#involved)
+
+
+<a name="agm"></a>
+# **Second Annual General Meeting and Election of Steering Committee Members**
+
+Our second [US-RSE Annual General Meeting (AGM)](https://us-rse.org/events/2021/2021-12-AGM/) will be held December 3rd, 2021, 1-3 pm ET, for all our members. Participants must register in advance [here](https://uasystem.zoom.us/meeting/register/tZYtce2pqzwuHNedu7cWhSQuvlqdvpXnLIMH). This will be an opportunity to hear from US-RSE leadership about our activities in 2021 and future plans for the Association. [Candidates](https://us-rse.org/2021-11-12-sc-candidates/) running for the US-RSE Steering Committee will be given an opportunity to introduce themselves. Questions or discussion topics can be submitted prior to the meeting [here](https://docs.google.com/forms/d/e/1FAIpQLSfaL7oM1hTLNS1Q60n08gWn7YFHMIwBR2BU3BFrtJDdxngw7A/viewform). All members are invited and encouraged to attend, and the meeting will be recorded and made available shortly afterwards for those who can’t make it.
+
+You can find [details on the election process of the US-RSE Steering Committee and timeline here](https://us-rse.org/about/election/)! Please don’t hesitate to ask any questions you might have on the [#election](https://usrse.slack.com/archives/C01BC66Q16E) channel on Slack. The voting process will be open from December 3-10, 2021, the election results will be announced by December 17, 2021, and newly elected Steering Committee members will take office January 1, 2022.
+
+<a name="survey"></a>
+# **International RSE Survey**
+
+As part of an international effort, US-RSE has worked with RSE organizations in other countries to develop the [2021 RSE Survey](https://softwaresaved.limequery.com/386272?lang=en). You don’t need to be called an RSE to complete the survey. Your formal job title does not matter: we want to hear from you if you develop software for academic research as part of your work. The survey provides considerable detail about the RSE role: a person’s route into the role, the type of work they conduct, their working practices, job satisfaction and demographics. This means it takes around 20 minutes to complete. We appreciate that this is a significant amount of your time to donate - and we thank you for it!
```

Comment:
> academic research

Industry R&D is excluded?

Reply:
Good point. This actually came up. As I understand it the survey has evolved from being targeted at UK academic institutions and has retained at least some of the language to be consistent. That said, I think we should remove the word academic here. It doesn't help.

Reply:
Removed.